spack.yaml

```
@@ -4,61 +4,52 @@
 # configuration settings.
 spack:
   specs:
-    - access-ram3@git.2024.08.0
+    - access-ram3@git.2025.08.000=null
   packages:
     # Direct dependencies
     um:
       require:
-        - '@13.0'
-
+        - '@13.5'
+        - model="vn13p5-rns"
+        - ~DR_HOOK
+        - +eccodes
     # Indirect dependencies
     eccodes:
       require:
-         - '@2.34.0'
-
+        - '@2.34.0'
```

```suggestion
        - '@2.34.0'
        - '+memfs'
```
Please add the `memfs` variant to [enable in-memory access to definitions](https://github.com/ACCESS-NRI/spack/blob/releases/v0.22/var/spack/repos/builtin/packages/eccodes/package.py#L85) to see if this will reduce our inode count.